README.md

```diff
@@ -1,3 +1,21 @@
+<!--
```

Comment:
Is it appropriate/required to add license in the README here? Doesn't show up in README in its default view. And we have already have a `LICENSE` which Github detects automatically (https://github.com/nginyc/rafiki/blob/master/LICENSE)

Reply:
i saw a comment by @nudles at https://github.com/nginyc/rafiki/pull/96 so I guess it is supposed to be added in README.md file

Reply:
If there is no side effect (e.g., the document generator cannot parse the comments), then just add the license in every file created by us.